docs/user-guide/services/operations.md

```diff
@@ -71,6 +75,10 @@ Required fields are:
     * `type`: Optional. If the type is `LoadBalancer`, sets up a [network load balancer](/docs/user-guide/load-balancer/)
       for your service. This provides an externally-accessible IP address that
       sends traffic to the correct port on your cluster nodes. 
+    * `loadBalancerSourceRanges:`: Optional. Must use with `LoadBalancer` type. 
+      If specified and supported by the platform, this will restrict traffic through the cloud-provider.
+      load-balancer will be restricted to the clients from specified IP ranges. This field will be ignored if the
+      cloud-provider does not support the feature. 
```

Comment:
If specified and supported by the cloud provider, this will restrict traffic such that the load balancer will be accessible only to clients from the specified IP ranges.